src/compiler/scala/tools/nsc/ast/TreeBrowsers.scala

```diff
@@ -279,6 +279,29 @@ abstract class TreeBrowsers {
         }
       )
       jmView add jmiCollapse
+      val jmiGoto = new JMenuItem(
+        new AbstractAction("Go to unit") {
+          putValue(Action.ACCELERATOR_KEY, KeyStroke.getKeyStroke(KeyEvent.VK_N, menuKey, false))
+          override def actionPerformed(actionEvent: ActionEvent) {
```

Comment:
If linting were on, we could better appreciate the pun on "go to unit:" within procedure syntax that omits `: Unit`.

Reply:
It's a last hurrah, of sorts. I assume that this PR means that @dwijnand can finally fix value classes for good.

Reply:
That's better than fixing them for worse.